src/types/config.js

```diff
@@ -84,6 +84,18 @@
  * @prop {import('./rpc').FocusUserInfo} [user]
  */
 
+/**
+ * List of theme elements which can be customized.
+ *
+ * @typedef {'accentColor'|
+ *   'annotationFontFamily' |
+ *   'appBackgroundColor'|
+ *   'ctaBackgroundColor'|
+ *   'ctaTextColor'|
+ *   'selectionFontFamily'
+ * } ThemeProperty
+ */
+
```

Comment:
Should this type be used in `src/sidebar/helpers/theme` (it defines its own `ThemeProperty` type).

Reply:
Ah, indeed. I'll update that module.